### PR TITLE
ICMSLST-1961 Reopen case email

### PR DESCRIPTION
**Pull request by marcuspp**

### CODE REVIEW

File: web/domains/case/views/views_search.py [hunk 1]
```diff
@@ -195,20 +196,21 @@ class ReopenApplicationView(
...
-        self.application.update_order_datetime()
-        self.update_application_status()
-        self.update_application_tasks()
-        document_pack.pack_draft_create(self.application)
-
-        messages.success(
-            request,
-            "The case has been reopened, and can be taken ownership of by any case manager.",
-        )
+        with transaction.atomic():
```
Comment (MattHolmes123) [inline web/domains/case/views/views_search.py:199]: The post method is already atomic:
`@method_decorator(transaction.atomic, name="post")`

Comment (MattHolmes123) [reply web/domains/case/views/views_search.py:199]: I don't really mind which we use but not both.

File: web/notify/constants.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import enum
+
+
+class DatabaseEmailTemplate(enum.Enum):
```
Comment (MattHolmes123) [inline web/notify/constants.py:4]: We are running ICMS using pythin 3.11.
Let's use StrEnum

```
>>> from enum import Enum, StrEnum
>>> class Foo(Enum):
...     VAL = "ASDF"
... 
>>> class Bar(StrEnum):
...     VAL = "ASDF"
... 
>>> print(Foo.VAL == "ASDF")
False
>>> print(Foo.VAL.value == "ASDF")
True
>>> print(Bar.VAL == "ASDF")
True
>>> print(Bar.VAL.value == "ASDF")
True
>>> 
```

**Review by MattHolmes123 — CHANGES_REQUESTED** [review]
added a few comments.

File: web/domains/case/views/views_search.py (hunk 1 above)
Comment (marcuspp) [reply web/domains/case/views/views_search.py]: 👍 I didn't notice the decorator so I have just removed the with statement

File: web/notify/constants.py (hunk 2 above)
Comment (marcuspp) [reply web/notify/constants.py]: Updated! I need to the read the whats new in 3.11 :-)

**Pull request merged**